test/Telegram.Bot.Tests.Integ/Admin Bot/ChannelAdminBotTests.cs

```diff
@@ -131,7 +132,7 @@ public ChannelAdminBotTests(TestsFixture testsFixture, Fixture classFixture)
         {
             await _fixture.SendTestCaseNotificationAsync(FactTitles.ShouldSetChatPhoto);
 
-            using (Stream stream = System.IO.File.OpenRead(Constants.FileNames.Photos.Logo))
+            using (FileStream stream = File.OpenRead(Constants.FileNames.Photos.Logo))
```

Comment:
Bot client methods take `Stream`. Again, the point is to make it clear what should be passed to the methods.

Reply:
I can make all of them Stream. Rn some were Stream and some FileStream